### PR TITLE
Jayasurya Shopping Cart Problem

### PR DESCRIPTION
**Pull request by jsuryakt**
Solved Shopping Cart Problem

### CODE REVIEW

File: Easy/1. Shopping Cart Problem/solutions/README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # Solutions
+![alt text](https://github.com/jsuryakt/code-with-girlscript-bangalore/blob/ShoppingCartEasy/Easy/1.%20Shopping%20Cart%20Problem/solutions/jsuryakt_Jayasurya/Screenshot%20(24).png?raw=true)
```
Comment (girlscript-blr) [inline Easy/1. Shopping Cart Problem/solutions/README.md]: These two lines are not required. Please don't modify any files outside your folder.

**Review by girlscript-blr — CHANGES_REQUESTED** [review]

File: Easy/1. Shopping Cart Problem/solutions/README.md (hunk 1 above)
Comment (jsuryakt) [reply Easy/1. Shopping Cart Problem/solutions/README.md]: Ok sorry 
I'll remove that.

**Review by girlscript-blr — APPROVED** [review]

**Comment by jsuryakt** [comment]
Thank you

**Comment by girlscript-blr** [comment]
Greetings from Code With Girlscript Bangalore😃

Dear participant, we have gone through your pull requests made in `EASY` sections of the problem statement.

Congratulations🎊 🎉  on getting your pull request successfully merged in the `Shopping Cart Problem - Easy` section in `code-with-girlscript-bangalore` repository. We will be distributing your valuable badges🥇🥈🥉🎖️ corresponding to the pull requests that are merged.

Learning is a never-ending process. 📈 Check out the [Intermediate](https://github.com/girlscript-blr/code-with-girlscript-bangalore/tree/master/Medium/1.%20Shopping%20Cart%20Problem) & [Advanced](https://github.com/girlscript-blr/code-with-girlscript-bangalore/tree/master/Hard/1.%20Shopping%20Cart%20Problem) level problem statements and try your hand at it for earning higher-level badges. Thanks for your contribution! Happy coding😃

**Pull request merged**